src/nrc/api/urls.py

```diff
@@ -18,16 +17,12 @@
         include(
             [
                 # API documentation
-                re_path(
-                    r"^schema/openapi(?P<format>\.json|\.yaml)$",
-                    SchemaView.without_ui(cache_timeout=settings.SPEC_CACHE_TIMEOUT),
-                    name="schema-json",
+                path(
+                    "schema/openapi.yaml", SpectacularAPIView.as_view(), name="schema"
```

Comment:
I wouldn't call it an advantage, but currently we do have it as two separate endpoints and keeping this change as is would remove the old endpoint (which could be a problem if anybody relies on this `/api/v1/schema/openapi.json` endpoint currently. In addition to that I think it would be good to keep this consistent between OF and the rest of the components as well. Also, having an endpoint end with the format `.yaml` and being able to ignore this yaml format by specifying `?format=json` seems a bit weird to me.